### DISCUSSION TITLE
Copilot doesn't work, no output to see why

### DISCUSSION THREAD

**Discussion by MetaBenji** [community/community#40223] [Copilot Conversations]
### Select Topic Area

Bug

### Body

For some reason Copilot ins't working on my Mac in VS Code.  Does anyone have any suggestions?

**What I've tried**
- restarting VS Code
- uninstalling and reinstalling copilot
- checking my Github account online to ensure I have it enabled.  I'm currently on a trial

**Environment**
- VS Code 1.73.1
- Copilot v1.61.7372 (github.copilot)

<img width="1462" alt="image" src="https://user-images.githubusercontent.com/101577156/204158068-c554265b-2db2-4b74-a5fd-6f226e18364c.png">

**Comment by ossanna16** [comment]
Hi there @MetaBenji and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by iamcjwilliams** [comment]
@MetaBenji Sorry for the trouble here. We experienced difficulties over the weekend that were our fault as we tried to roll out some new features. Please accept my apologies on behalf of the wider Copilot team. Are you still experiencing issues? Are you able to provide a screenshot of an error you're receiving?

**Reply by MetaBenji** [reply]
No worries @iamcjwilliams , stuff happens!  I don't see any specific error.  When I check the output tab in the VS Code terminal and filter for just github copilot, it's completely blank.  Is that to be expected?

In my `settings.json`, I have it enabled for *, which I assume is all languages and I also have it enabled for `typescript` and `typescriptreact`.  

I'm testing by going into a `.ts` or `.tsx` file and type `function addTwoNumbers(one: number, two: number)` I would expect it to suggest the body of the function.

Terminal Output
<img width="1468" alt="image" src="https://user-images.githubusercontent.com/101577156/204920164-98d3d40a-2c61-4a54-b7c0-13272afdc33e.png">

Settings
<img width="358" alt="image" src="https://user-images.githubusercontent.com/101577156/204920372-7b7be33c-8012-462f-b8c4-8671fd5c818d.png">

**Reply by iamcjwilliams** [reply]
@MetaBenji Have you tried different functions and still received a blank screen? Also, have you tried to hover over your comment function to open copilot? 

<img width="478" alt="Screenshot 2022-12-01 at 3 17 24 PM" src="https://user-images.githubusercontent.com/40559511/205160999-fe8566ba-8c7e-4cae-aa4d-72c5d9bce6ca.png">

**Reply by MetaBenji** [reply]
Yea, I've tried different functions and still don't get any suggestions. 

Questions
1.  Should I be seeing anything in the output section in the built in terminal?
2. What's the comment function?  I'm not sure what that is?

**Reply by MetaBenji** [reply]
I don't see anything copilot suggestions at all in VS Code

**Reply by MetaBenji** [reply]
I managed to fix it!  I had to sign out and sign back in with Github
